indexing/src/main/scala/org/allenai/common/indexing/BuildCorpusIndex.scala

```diff
@@ -36,7 +38,13 @@ class BuildCorpusIndex(config: Config) extends Logging {
 
   /** Get Index Name and Index Type. */
   val esConfig: Config = config[Config]("elasticSearch")
-  val indexName: String = esConfig[String]("indexName")
+  val indexName: String = {
+    // The index name must be appended with the current date.
+    val name = esConfig[String]("indexName")
+    val dateFormat = new SimpleDateFormat("yyyy-MM-dd")
+    name + "-" + dateFormat.format(Calendar.getInstance().getTime())
```

Comment:
At current writing, that workflow would fail on the second attempt due to the index already existing.  It needs to be deleted manually.  I have a long-term task to rework this entire tool (which would delete a failed index), but I want to keep this change minimally invasive.

Hour/minute/second is a bit noisy, and I'd actually like to discourage extremely frequent (e.g. same day) reindexing, since we need to keep copies of any index that's been used for repeatability.